Dockerfile

```diff
@@ -56,6 +56,8 @@ RUN mkdir -p /home/${DOCKER_USER}/devel_ws && chown ${DOCKER_USER}:${DOCKER_USER
 RUN echo "source /opt/ros/$ROS_DISTRO/setup.bash" >> /etc/bash.bashrc
 RUN echo "PS1='\[\033[01;35m\]ros-$ROS_DISTRO@devel\[\033[00m\]:\[\033[01;34m\]\w\[\033[00m\]\$ '" >> /etc/skel/.bashrc
 
+ENV GPG_TTY=$(tty)
```

Comment:
Why setting this env here, and not directly at docker-compose.yml?

Reply:
I couldn't access the `tty` command there